retrieverdash/dashboard_script/dashboard_script.py

```diff
@@ -20,20 +20,21 @@
                     'butterfly-population-network',
                     'partners-in-flight',
                     'portal']
-try:
-    dataset_detail = json.load(open('dataset_details.json', 'r'))
-except IOError:
-    with open("dataset_details.json", 'w') as json_file:
-        dataset_detail = dict()
-        json.dump(dataset_detail, json_file)
 
 
+dataset_detail=None
```

Comment:
dataset_detail=None
dataset_detail = None
I think we should not have this here, it is hanging and not needed